pkg/util/util_test.go

```diff
@@ -65,34 +65,40 @@ func TestThroughputCalc(t *testing.T) {
-		"password='foo bar' host='localhost'":                                "password='****' host='localhost'",
-		"Password='foo bar' host='localhost'":                                "password='****' host='localhost'",
-		"password='foo'bar' host='localhost'":                                "password='****'bar' host='localhost'",
-		"password= 'foobar' host='localhost'":                                "password= '****' host='localhost'",
-		"password=  'foobar' host='localhost'":                               "password=  '****' host='localhost'",
-		"pass='foobar' host='localhost'":                                     "pass='foobar' host='localhost'",
-		"host='localhost' password='foobar'":                                 "host='localhost' password='****'",
-		"password:foobar  host: localhost":                                   "password:****  host: localhost",
-		"password:foo bar  host: localhost":                                  "password:****  host: localhost",
-		"password: foobar  host: localhost":                                  "password: ****  host: localhost",
-		"password:  foobar  host: localhost":                                 "password:  ****  host: localhost",
-		"password:  foobar with spaces host: localhost":                      "password:  **** host: localhost",
-		"password: foobar with spaces host: localhost":                       "password: **** host: localhost",
-		"password:foobar with spaces host: localhost":                        "password:**** host: localhost",
-		"password:\"foo bar\" host: localhost":                               "password:**** host: localhost",
-		"Password: \"foo bar\" host: localhost":                              "password: **** host: localhost",
-		"pass:foobar host: localhost":                                        "pass:foobar host: localhost",
-		"host: localhost password: foobar":                                   "host: localhost password: ****",
-		"host: localhost Password: foobar":                                   "host: localhost password: ****",
-		"postgres://postgres:password@localhost:5432/postgres?sslmode=allow": "postgres://postgres:****@localhost:5432/postgres?sslmode=allow",
+		"password='foobar' host='localhost'":                                                   "password='****' host='localhost'",
+		"password='foobar&' host='localhost'":                                                  "password='****' host='localhost'",
+		"password='foo bar' host='localhost'":                                                  "password='****' host='localhost'",
+		"Password='foo bar' host='localhost'":                                                  "password='****' host='localhost'",
+		"password='foo'bar' host='localhost'":                                                  "password='****'bar' host='localhost'",
+		"password= 'foobar' host='localhost'":                                                  "password= '****' host='localhost'",
```

Comment:
Is space before '* necessary in masking? Or is it the restrain due to implement method?

Reply:
Respects previous spacing.